sdk/deviceupdate/azure-iot-deviceupdate/samples/simple_sample.py

```diff
@@ -0,0 +1,56 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
```

Comment:
I think something like "sample_hello_world" is better than "simple_sample", afaik we use "hello_world" more to showcase the basic sample

Reply:
This heading also doesn't look right, can you follow the heading here? https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/textanalytics/azure-ai-textanalytics/samples/sample_detect_language.py#L1-L24